### PR TITLE
Changelog for MDCB 2.0.4 release

### PR DESCRIPTION
**Pull request by tbuchaillot**
Adding changelog logs for MDCB 2.0.4 release

https://github.com/TykTechnologies/tyk-sink/releases/tag/v2.0.4

### CODE REVIEW

File: tyk-docs/content/release-notes/mdcb/mdcb.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ menu:
 weight: 255
 url: "/release-notes/mdcb"
 ---
+## 2.0.4
```
Comment (letzya) [inline tyk-docs/content/release-notes/mdcb/mdcb.md:8]: Looks like this is a v2 page. Can you please rename the file and title and parent to match the format of the gateway and dashboard?  (we'd need also an alias for the url)
image

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/release-notes/mdcb/mdcb.md (hunk 1 above)
Comment (tbuchaillot) [reply tyk-docs/content/release-notes/mdcb/mdcb.md:8]: @letzya as I understand, it's a whole changelog for the component - it's not a page for this version.

Comment (tbuchaillot) [reply tyk-docs/content/release-notes/mdcb/mdcb.md:8]: if we rename it - the MDCB 2.0.4 page will have all the previous release notes too. It doesn't make sense to me tbh.

Comment (letzya) [reply tyk-docs/content/release-notes/mdcb/mdcb.md:8]: we have pages per major and minor releases but not patches. On this page, we have only patches. the next release with 2.1 will need a new page so I'm suggesting this page be renamed to `MDCB v2.0`

Comment (tbuchaillot) [reply tyk-docs/content/release-notes/mdcb/mdcb.md:8]: now I get it! I'll update

Comment (tbuchaillot) [reply tyk-docs/content/release-notes/mdcb/mdcb.md:8]: image
👌

**Review by andyo-tyk — COMMENTED** [review]
Thanks Tomas - and good suggestion Yaara (the previous page was pulled together by Leo as we had nothing at all on Tyk.io before in terms of release notes for MDCB!).

**Review by andyo-tyk — APPROVED** [review]

**Review by letzya — APPROVED** [review]

**Review by letzya — APPROVED** [review]

**Pull request merged**